test/Dense.hs

```diff
@@ -281,5 +281,4 @@ gcdExtTests = localOption (QuickCheckMaxSize 12) $ testGroup "gcdExt"
 sameUpToUnits :: (Eq a, GcdDomain a) => a -> a -> Bool
 sameUpToUnits x y = x == y ||
   isJust (x `divide` y) && isJust (y `divide` x)
-
```

Comment:
I believe this is a merge artifact?

Reply:
Yeah - I tried to make the spacing consistent with the rest of the module.